.github/workflows/windows.yml

```diff
@@ -51,6 +51,21 @@ jobs:
       with:
         url: "https://download.sysinternals.com/files/Coreinfo.zip"
         target: _build/coreinfo/
+    - name: Gather openssl version info
+      working-directory: _build
+      run: |
+        apps/openssl.exe version -v
+        apps/openssl.exe version -v | awk '{print $2}'
+        apps/openssl.exe version -v | awk '{print $2}' | sed -e's/-.*$//'
+        echo "OSSL_VERSION=$(apps/openssl.exe version -v | awk '{print $2}' | sed -e's/-.*$//')" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf8 -Append
```

Comment:
Doesn't this need amending to remove the patch number? (and why doesn't this issue cause CIs to fail??)

Reply:
ugh...its passing because we had to write the windows_registry_test such that it gets skipped if the registry keys aren't set (for local builds where we didn't want to modify the user system).  thats really irritating